src/components/Grid/Grid.tsx

```diff
@@ -8,13 +8,17 @@ import {
   UIComponentProps,
   ChildrenComponentProps,
   commonPropTypes,
+  ContentComponentProps,
 } from '../../lib'
-import { Extendable, ShorthandValue } from '../../../types/utils'
+import { Extendable } from '../../../types/utils'
 import { Accessibility } from '../../lib/accessibility/types'
 import { defaultBehavior } from '../../lib/accessibility'
 import ReactNode = React.ReactNode
 
-export interface GridProps extends UIComponentProps, ChildrenComponentProps {
+export interface GridProps
+  extends UIComponentProps,
+    ChildrenComponentProps,
+    ContentComponentProps<React.ReactNode | React.ReactNode[]> {
```

Comment:
seems that the same logic could be applied to many other components, like `AccordionContent`, `AccordionTitle`, `ButtonGroup`

Reply:
yes, actually, would agree with that - now see the reason. In contrast to those components I've mentioned, `Grid` actually, by design, expects that multiple elements may be passed as its content